kfdefs/base/google-spark-operator/kfdef.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: guimou
+          path: google-spark-operator/operator
+      name: google-spark-operator
+  repos:
+    - name: manifests
```

Comment:
I don't think we need the "manifests" repo here since it is not being used in the kfdef

Reply:
the eventual goal is to switch to this repo

Reply:
I don;t mid either way. The fact is that keeping it in here makes this manifest eligible for ODH z-stream update PRs  which makes us keep an eye on this manifest more closely. The fact that there's an open PR for this code makes me assume that in 1 or 2 z-stream releases we will be able to switch over to ODH upstream repo here.